### DISCUSSION TITLE
FTP into LXC containers?

### DISCUSSION THREAD

**Discussion by johndoe0815** [tteck/Proxmox#917] [Help Needed]
Hello,
is it somehow possible to access the LXC container's from tteck content via FTP?
I tried to connect to its IP address with WINSCP, but it is asking for user and password. No credentials does not work, but I haven't set up any with the tteck scripts.
Trilium is broken after a power loss of the proxmox hosts, only having full CPU usage but not starting up properly. I wanted to access the automatic weekly backup database to transfer that to a new machine, because my last container backup is from a longer time ago.

**Comment by Vakrehus** [comment]
Don't do FTP, use SSH with WinSCP.

https://github.com/tteck/Proxmox/discussions/385

**Reply by johndoe0815** [reply]
According to https://winscp.net/eng/docs/free_ssh_client_for_windows, SFTP is access via SSH. I added a root password and allowed login as per the docu, but still get access denied then with WinSCP (with the same user root and its password that works to access the console in proxmox).
